src/AssetPack/AssetPack.router.ts

```diff
@@ -134,33 +135,42 @@ export class AssetPackRouter extends Router {
 
     let assetPacks: FullAssetPackAttributes[] = []
 
-    // Get default asset packs
-    if (!ethAddress || ethAddress !== DEFAULT_ETH_ADDRESS) {
-      const defaultAssetPacks = await this.logExecutionTime(
-        this.retrieveDefaultAssetPacks,
+    // Get user asset packs
+    if (ethAddress) {
+      const userAssetPacks = await this.logExecutionTime(
+        () => AssetPack.findByEthAddressWithAssets(ethAddress),
         `Get the user\'s (${ethAddress}) asset packs`,
         tracer
       )
-      assetPacks = [...defaultAssetPacks]
-      this.logger.info(
-        `[${tracer}] Assets pack length after adding the default asset packs: ${assetPacks.length}`
-      )
+
+      // If the user doesn't have any asset packs, return the default asset packs
+      if (userAssetPacks.length === 0) {
+        return this.sendDefaultAssetPacksRaw(res, tracer)
+      }
+
+      assetPacks = [...assetPacks, ...userAssetPacks]
+    } else {
+      return this.sendDefaultAssetPacksRaw(res, tracer)
     }
 
-    // Get user asset packs
-    if (ethAddress) {
-      const userAssetPacks = await this.logExecutionTime(
-        () => AssetPack.findByEthAddressWithAssets(ethAddress),
+    // Get default asset packs
+    if (ethAddress !== DEFAULT_ETH_ADDRESS) {
+      const defaultAssetPacks = await this.logExecutionTime(
+        this.retrieveDefaultAssetPacks,
         'Get the default asset packs',
         tracer
       )
-      assetPacks = [...assetPacks, ...userAssetPacks]
+      assetPacks = [...assetPacks, ...defaultAssetPacks]
+      this.logger.info(
+        `[${tracer}] Assets pack length after adding the default asset packs: ${assetPacks.length}`
+      )
     }
 
     this.logger.info(
       `[${tracer}] Final assets pack length: ${assetPacks.length}`
     )
-    return assetPacks
+
+    return res.json({ ok: true, data: assetPacks })
```

Comment:
we can use [server.sendOk](https://github.com/decentraland/decentraland-server/blob/master/src/server/index.ts#L76) here! and maybe [here](https://github.com/decentraland/builder-server/pull/285/files#diff-d4ee7ab7527de0ddae7f11f2a1cd1bcfebc1ab0c2550ae5b878b1c9d741e8cc8R322)

Reply:
Done!